starfish/image/_filter/laplace.py

```diff
@@ -108,7 +109,8 @@ def run(
         verbose : bool
             if True, report on filtering progress (default = False)
         n_processes : Optional[int]
-            Number of parallel processes to devote to calculating the filter
+            Number of parallel processes to devote to calculating the filter. If None, defaults to
```

Comment:
Filters aren't calculated, they're defined by a parameter :). "Number of parallel process to devote performing the filter operation" or something like that. This bug appears elsewhere in the codebase, maybe try a find and replace?